docs/pages/2020_Intro_Week/index.rst

```diff
@@ -23,11 +23,75 @@ This is the content of the `AiiDA virtual tutorial <http://www.aiida.net/aiida-v
++                    +--------------------------------------------------------------------+
+|                    | `Setting up an environment with Quantum Mobile Cloud`_             |
++--------------------+--------------------------------------------------------------------+
+| Wednesday July 8th | `Provenance and workflows in AiiDA`_                               |
++--------------------+--------------------------------------------------------------------+
+| Thursday July 9th  | `The AiiDA Plugin Ecosystem`_                                      |
++--------------------+--------------------------------------------------------------------+
+| Friday July 10th   | `The Materials Cloud`_                                             |
++                    +--------------------------------------------------------------------+
+|                    | `AiiDA lab`_                                                       |
++                    +--------------------------------------------------------------------+
+|                    | `AiiDA and OPTIMADE`_                                              |
++                    +--------------------------------------------------------------------+
+|                    | `OSSCAR`_                                                          |
++--------------------+--------------------------------------------------------------------+
+
+.. _AiiDA Tutorial Overview: https://www.youtube.com/watch?v=RtdXXWFLvF8&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+.. _Introduction to AiiDA: https://www.youtube.com/watch?v=jigMCyWGNAE&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+.. _Setting up an environment with Quantum Mobile Cloud: https://www.youtube.com/watch?v=vlmjVwGJgEU&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+.. _Provenance and workflows in AiiDA: https://www.youtube.com/watch?v=KpiLIA8ge1w&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+.. _The AiiDA Plugin Ecosystem: https://www.youtube.com/watch?v=bjTUnHXZ6oY&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+.. _The Materials Cloud: https://www.youtube.com/watch?v=_T8HBuOYUyc&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+.. _AiiDA lab: https://www.youtube.com/watch?v=BwtxyTAVugY&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+.. _AiiDA and OPTIMADE: https://www.youtube.com/watch?v=F0e3cQHNbNM&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+.. _OSSCAR: https://www.youtube.com/watch?v=3rkhzYBUuA4&list=PL19kfLn4sO_-e_A9lVYb_NBNcwoVvUP6V
+
+Next to these presentations from the ``aiida-core`` developers, there are also the contributed talks from the plugin developers.
+These can be watched at any time, but please try to watch them **before Friday July 10th**, as we have planned a Q&A with the plugin developers at the start of the first hands-on session of that day.
+Here is a list of all the contributed talks:
+
++--------------------------+----------------------------------------------------------------------------------------------------------------------+
```

Comment:
Add a header line.
Maybe it would be better to do it like this for the presentations above as well. Adding the presenter as a separate column, but then as the last column?

Reply:
Also, you should put the proper title for all people here (maybe also for us).

Reply:
Header lines added!

I added the titles for the plugin developers (afaik they are all Dr.'s). I didn't do so for us, as the table is already too big to fit on the page.

Reply:
I think either you do it for all or for noone?

Reply:
And are you sure they're all PhDs?

Reply:
I checked this quickly when I was composing the list of contributions for Marco. Let's just remove the titles for now, I can add them again later if we can wrap the text in the final column.